native-tls/src/tls_client_connection.rs

```diff
@@ -1,104 +0,0 @@
-// Copyright (C) 2015 Benjamin Fry <benjaminfry@me.com>
```

Comment:
TLSClientConnection for native-tls and openssl are being removed, as they can't be `Send+Sync`. The recommendation is to use Rustls or to use the tokio variants directly if openssl and/or nativetls are desired.